Mage.Sets/src/mage/sets/JudgeGiftCards2014.java

```diff
@@ -0,0 +1,38 @@
+package mage.sets;
+
+import mage.cards.ExpansionSet;
+import mage.constants.Rarity;
+import mage.constants.SetType;
+
+/**
+ * https://scryfall.com/sets/j14
+ */
+public class JudgeGiftCards2014 extends ExpansionSet {
+
+    private static final JudgeGiftCards2014 instance = new JudgeGiftCards2014();
+
+    public static JudgeGiftCards2014 getInstance() {
+        return instance;
+    }
+
+    private JudgeGiftCards2014() {
+        super("Judge Gift Cards 2014", "J14", ExpansionSet.buildDate(2014, 1, 1), SetType.PROMOTIONAL);
+        this.hasBoosters = false;
+        this.hasBasicLands = true;
+
+        cards.add(new SetCardInfo("Elesh Norn, Grand Cenobite", 8, Rarity.RARE, mage.cards.e.EleshNornGrandCenobite.class));
+        cards.add(new SetCardInfo("Force of Will", 4, Rarity.RARE, mage.cards.f.ForceOfWill.class));
+        cards.add(new SetCardInfo("Forest", "5★", Rarity.LAND, mage.cards.basiclands.Forest.class));
+        cards.add(new SetCardInfo("Greater Good", 2, Rarity.RARE, mage.cards.g.GreaterGood.class));
+        cards.add(new SetCardInfo("Hanna, Ship's Navigator", 5, Rarity.RARE, mage.cards.h.HannaShipsNavigator.class));
+        cards.add(new SetCardInfo("Island", "2★", Rarity.LAND, mage.cards.basiclands.Island.class));
+        cards.add(new SetCardInfo("Karador, Ghost Chieftain", 1, Rarity.RARE, mage.cards.k.KaradorGhostChieftain.class));
+        cards.add(new SetCardInfo("Mountain", "4★", Rarity.LAND, mage.cards.basiclands.Mountain.class));
+        cards.add(new SetCardInfo("Nekusar, the Mindrazer", 7, Rarity.RARE, mage.cards.n.NekusarTheMindrazer.class));
+        cards.add(new SetCardInfo("Oloro, Ageless Ascetic", 9, Rarity.RARE, mage.cards.o.OloroAgelessAscetic.class));
+        cards.add(new SetCardInfo("Plains", "1★", Rarity.LAND, mage.cards.basiclands.Plains.class));
+        cards.add(new SetCardInfo("Riku of Two Reflections", 3, Rarity.MYTHIC, mage.cards.r.RikuOfTwoReflections.class));
+        cards.add(new SetCardInfo("Swamp", "3★", Rarity.LAND, mage.cards.basiclands.Swamp.class));
```

Comment:
Don't use unicode symbols in card numbers or names, use only ascii (there are some regexp with save/load deck files -- it can fail with that card numbers chars).

Reply:
Ok, This was copied from Scryfall as-is.
Then I’ll remove the ★ and add a special download rule.

Reply:
This may be a nice addition to the card data verification test.
It should be easy to test the card names and numbers against the RE `[a-zA-Z0-9'_"]+` or similar.

Reply:
Replaced the ★.